src/main/scala/firrtl/annotations/Annotation.scala

```diff
@@ -165,3 +166,5 @@ object Annotation
 case class DeletedAnnotation(xFormName: String, anno: Annotation) extends NoTargetAnnotation {
   override def serialize: String = s"""DELETED by $xFormName\n${anno.serialize}"""
 }
+
```

Comment:
can this have scaladoc added?

Reply:
I could but shouldn't an unrelated tests like this go in a separate PR?

Reply:
@chick I think you may have responded to the wrong cmoment

Reply:
@jackkoenig @mwachs5 I meant to say, adding scaladoc to pre-existing code that is unrelated to this PR should go on a separate PR. But I'm happy to add some. Any thoughts on what it should say, I feel like DeletedAnnotations are heading towards deprecation and using them, particularly looking inside them, is an anti-pattern